tests/Makefile

```diff
@@ -0,0 +1,16 @@
+# Just a Makefile for automatic testing
+.PHONY: all
+
+all: clean init test
+
+clean:
+	rm -rf .jenkinsfile-runner-test-framework
```

Comment:
Since you're testing CWP, why don't you name it as .cwp or similar?

Reply:
The test framework is checked out there so that it can be used, that's why it has that name.

Reply:
You're right. For a moment my mind went blank and I forgot the git clone execution before the sync one